src/Microsoft.Android.Build.BaseTasks/Files.cs

```diff
@@ -157,6 +229,33 @@ public static bool CopyIfBytesChanged (byte[] bytes, string destination)
 		}
 
 		public static bool CopyIfStreamChanged (Stream stream, string destination)
+		{
+			int retryCount = 0;
+			int attempts = GetFileWriteRetryAttempts ();
+			int delay = GetFileWriteRetryDelay ();
+			while (retryCount <= attempts) {
+				try {
+					return CopyIfStreamChangedOnce (stream, destination);
+				} catch (Exception e) {
+					switch (e) {
+						case UnauthorizedAccessException:
+						case IOException:
+							int code = Marshal.GetHRForException (e);
+							if ((code != ERROR_ACCESS_DENIED && code != ERROR_SHARING_VIOLATION) || retryCount == attempts) {
```

Comment:
For general safety, should we do this to prevent an infinite loop:
```suggestion
							if ((code != ERROR_ACCESS_DENIED && code != ERROR_SHARING_VIOLATION) || retryCount >= attempts) {
```